### PR TITLE
Styled Content Key Page

### PR DESCRIPTION
**Pull request by KaydenML**
@mrferris 
After Image before using bootstrap
image

Before Image
image

### CODE REVIEW

**Comment by pipermerriam** [comment]
The site should have bootstrap on it.  Maybe this can be done with the existing card styling from bootstrap?

https://getbootstrap.com/docs/5.2/components/card/

**Comment by KaydenML** [comment]
@pipermerriam sounds good thanks for the feedback. I'm going to look into styling these pages with bootstrap instead

**Comment by KaydenML** [comment]
@pipermerriam sounds good thanks for the feedback. I'm going to look into styling these pages with bootstrap instead

**Comment by KaydenML** [comment]
After Image, after being styled using bootstrap
image

File: glados-web/templates/contentkey_detail.html [hunk 1]
```diff
@@ -2,32 +2,47 @@
 
 {% block title %}Content Key: {{ content_key_model.key_as_hex() }}{% endblock %}
 
+{% block head %}
+<link href="/static/css/glados_pages.css" rel="stylesheet">
```
Comment (pipermerriam) [inline glados-web/templates/contentkey_detail.html:6]: This looks like it's maybe leftover?

**Review by pipermerriam — APPROVED** [review]

File: glados-web/templates/contentkey_detail.html (hunk 1 above)
Comment (KaydenML) [reply glados-web/templates/contentkey_detail.html]: fixed

**Pull request closed without merging**